README.md

```diff
@@ -85,932 +119,667 @@ Unlike many authorization systems where a service controls access to resources i
+# Canonicalization
 
-### 6.2.2 Token Uniqueness
+## Cryptosuite
 
-Each remote invocation MUST be a unique UCAN: for instance using a nonce (`nnc`) or simply a unique expiry. The recipient MUST validate that they have not received the top-level UCAN before. For implementation recommentations, please refer to the [replay attack prevention] section. 
+Across all UCAN specifications, the following cryptosuite MUST be supported:
 
-## 6.3 Proof Chaining
+| Role      | REQUIRED Algorithms               | Notes                                | 
+|-----------|-----------------------------------|--------------------------------------|
+| Hash      | [SHA-256]                         |                                      |
+| Signature | [Ed25519], [P-256], [`secp256k1`] | Preference of Ed25519 is RECOMMENDED |
+| [DID]     | [`did:key`]                       |                                      |
 
-Each capability MUST either be originated by the issuer (root capability, or "parenthood") or have one-or-more proofs in the `prf` field to attest that this issuer is authorized to use that capability ("introduction"). In the introduction case, this check MUST be recursively applied to its proofs until a root proof is found (i.e. issued by the resource owner).
+## Encoding
 
-Except for rights amplification (below), each capability delegation MUST have equal or narrower capabilities from its proofs. The time bounds MUST also be equal to or contained inside the time bounds of the proof's time bounds. This lowering of rights at each delegation is called "attenuation."
+All UCANs MUST be canonically encoded with [DAG-CBOR] for signing. A UCAN MAY be presented or stored in other [IPLD] formats (such as [DAG-JSON]), but converted to DAG-CBOR for signature validation.
 
-## 6.4 Rights Amplification
+## Content Identifiers
 
-Some capabilities are more than the sum of their parts. The canonical example is a can of soup and a can opener. You need both to access the soup inside the can, but the can opener may come from a completely separate source than the can of soup. Such semantics MAY be implemented in UCAN capabilities. This means that validating particular capabilities MAY require more than one direct proof. The relevant proofs MAY be of a different resource and ability from the amplified capability. The delegated capability MUST have this behavior in its semantics, even if the proofs do not.
+A UCAN token MUST be configured as follows:
 
-## 6.5 Content Identifiers
+| Parameter    | REQUIRED Configuration |
+|--------------|------------------------|
+| Version      | [CIDv1]                | 
+| [Multibase]  | [`base58btc`]          | 
+| [Multihash]  | [SHA-256]              |
+| [Multicodec] | [DAG-CBOR]             |
 
-A UCAN token MUST be referenced as a [base32] [CIDv1]. [SHA2-256] is the RECOMMENDED hash algorithm.
+> [!NOTE]
+> All CIDs encoded as above start with the characters `zdpu`.
```

Comment:
If they're canonically DAG-CBOR why is the multibase needed, as opposed to [staying in binary](https://github.com/multiformats/cid?tab=readme-ov-file#how-does-it-work)? Maybe I've been talking to the wrong people but I thought CIDv1 was _canonically_ binary and _optionally_ multibased, so it feels like an extra roundtrip to multibase it and then drop back down to CBOR...